src/Compilers/CSharp/Portable/Symbols/Source/SourceNamespaceSymbol.AliasesAndUsings.cs

```diff
@@ -320,7 +320,7 @@ private MergedGlobalAliasesAndUsings GetMergedGlobalAliasesAndUsings(ConsList<Ty
                                         {
                                             if (!uniqueUsings.Add(namespaceOrType.NamespaceOrType))
                                             {
-                                                diagnostics.Add(ErrorCode.HDN_DuplicateWithGlobalUsing, namespaceOrType.UsingDirective!.Name.Location, namespaceOrType.NamespaceOrType);
+                                                diagnostics.Add(ErrorCode.HDN_DuplicateWithGlobalUsing, namespaceOrType.UsingDirective!.Type.Location, namespaceOrType.NamespaceOrType);
```

Comment:
the changes in this file are great.  ALl the code could already handle just passing a type along instead of a name.